corehq/apps/app_manager/templates/app_manager/form_designer.html

```diff
@@ -18,6 +18,17 @@
       padding-bottom: 0;
       margin-bottom: 0;
     }
+    .custom-button-style {
```

Comment:
first, adding styles to in-page styles is not good practice.
second, the name of this should be more descriptive.

Reply:
please move custom styles to `app_manager.less`

Reply:
Moved to form_editing.less: b9f4892238fde093553a0982b960ed8f12168c6a